packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -919,4 +936,12 @@ class AndroidNavigationDelegate extends PlatformNavigationDelegate {
   ) async {
     _onWebResourceError = onWebResourceError;
   }
+
+  /// Invoked when a render process has exited.
+  Future<void> setOnRenderProcessGone(
+    RenderProcessGoneCallback onRenderProcessGone,
```

Comment:
Since this is the only use of `RenderProcessGoneCallback`, you should use an inline function type instead: https://dart.dev/guides/language/effective-dart/design#prefer-inline-function-types-over-typedefs

Reply:
Changed callback to be inline function.